exercises/go-2-a/.meta/start_1.go

```diff
@@ -0,0 +1,34 @@
+package go2a
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func concat(description string, str string) string {
+	if description != "" {
+		return description + " or " + str
+	}
+
+	return description + str
+}
+
+func Describe(amount int) string {
```

Comment:
Oh... and maybe we should add comments to exported identifiers... just to make sure people don't get the idea we don't care about them.

Reply:
just run lint ?